jupyterlab_bigquery/src/components/query_editor/query_text_editor/query_text_editor.tsx

```diff
@@ -67,7 +69,6 @@ interface QueryRequestBodyType {
 
 const SQL_EDITOR_OPTIONS: editor.IEditorConstructionOptions = {
   lineNumbers: 'on',
-  automaticLayout: true,
```

Comment:
This was removed and replaced by a window resize listener, since this option actually calls a check every few milliseconds to see if the window was resized, which supposedly can slow the editor down.